nixos/modules/services/mail/davmail.nix

```diff
@@ -58,18 +58,26 @@ in
 
     config = mkIf cfg.enable {
 
-      services.davmail.config.davmail = mapAttrs (name: mkDefault) {
-        server = true;
-        disableUpdateCheck = true;
-        logFilePath = "/var/log/davmail/davmail.log";
-        logFileSize = "1MB";
-        mode = "auto";
-        url = cfg.url;
-        caldavPort = 1080;
-        imapPort = 1143;
-        ldapPort = 1389;
-        popPort = 1110;
-        smtpPort = 1025;
+      services.davmail.config = mapAttrs (name: mkDefault) {
+        davmail = {
```

Comment:
You need to have `mkDefault` one each individual setting. How it is now, the `mkDefault` will only be on `davmail` and `log4j`, meaning if you assign `services.davmail.config.davmail.*`, it will clear all these defaults.

Reply:
@Infinisil good to go?